share/spack/spack-completion.bash

```diff
@@ -533,7 +659,7 @@ _spack_env_view () {
     then
         compgen -W "-h --help" -- "$cur"
     else
-        compgen -W "regenerate enable disable" -- "$cur"
+        :
```

Comment:
This got covered up by the last commit, but the positional metavar here is "{regenerate,enable,disable}". I could hardcode things to replace `{regenerate,enable,disable}` with `regenerate enable disable`, but the more robust option would be to re-implement this as subsubcommands.